frontend/dockerfile/docs/experimental.md

```diff
@@ -138,3 +138,19 @@ $ buildctl build --frontend=dockerfile.v0 --local context=. --local dockerfile=.
 You can also specify a path to `*.pem` file on the host directly instead of `$SSH_AUTH_SOCK`.
 However, pem files with passphrases are not supported.
 
+### RUN --ents=security.insecure
+
+This runs the command without sandbox in insecure mode, which allows to run flows requiring elevated
+privileges (e.g. containerd).
```

Comment:
You can mention that this is equivalent to `docker run --privileged`

Reply:
thanks, done